docs/hip-16.md

```diff
@@ -63,29 +56,44 @@ do not autorenew, and are always removed from the ledger when they expire.
 
 ## Backwards Compatibility
 
-There is no change in existing protobufs. Account and entity owners must ensure that linked autorenew and admin accounts have
-sufficient balances for autorenewal fees, or risk permanent removal of their entity! The Hedera Product team will set and
-publicize the timeline for enabling the autorenewal and autoremoval behaviors.
+There is no change in existing protobufs. Account and entity owners must ensure that linked autorenew and admin accounts have sufficient balances for autorenewal fees, or risk permanent removal of their entity.
+
+Every account will receive one free auto renewal at implementation of this feature. This will have the effect of extending the initial period for autorenewal ~92 days.
 
 ## Security Implications
 
-N/A
+A Hedera Account without an hbar balance sufficient to cover the cover the cost of renewal at the point of renewal would be marked for deletion. The Account owner then have a grace period to fund the account and renewal before permanent deletion.
+
+If the autoRenewAccount of a topic does not have sufficient balance the topic would be deleted. The ledger cannot enforce agreements regarding funding of the topic made by participants in the topic. 
+
+Any entity can have its expiration date extended by anyone. Not just the admin key. They expiration date is the only field that can be changed in an update without signature by the owner or the admin. 
+
+Entities who leverage omnibus accounts for services including wallets, exchanges, and custody will need to account for the deduction of hbar from any Hedera Accounts used in their system at time of autorenewal.
```

Comment:
Accounts who leverage omnibus entities for services including wallets, exchanges, and custody will need to account for the deduction of hbar from any Hedera Entities used in their system at time of autorenewal.